data/en/census_individual.json

```diff
@@ -2775,6 +2775,118 @@
             },
             {
                 "type": "Questionnaire",
+                "id": "hours-worked",
+                "title": "{{ [answers.first_name, answers.last_name] | format_household_name }}",
+                "description": "",
+                "questions": [{
+                    "title": "In your main job, how many hours a week do you usually work?",
+                    "id": "hours-worked-question",
+                    "type": "General",
+                    "answers": [{
+                        "id": "hours-worked-answer",
+                        "mandatory": true,
+                        "guidance": {
+                            "show_guidance": "Show further guidance",
+                            "hide_guidance": "Hide further guidance",
+                            "content": [{
+                                "description": "Include paid and unpaid overtime"
+                            }]
+                        },
+                        "options": [{
+                                "label": "15 or less",
+                                "value": "15 or less"
+                            },
+                            {
+                                "label": "16 - 30",
+                                "value": "16 - 30"
+                            },
+                            {
+                                "label": "31 - 48",
+                                "value": "31 - 48"
+                            },
+                            {
+                                "label": "49 or more",
+                                "value": "49 or more "
+                            }
+                        ],
+                        "type": "Radio"
+                    }]
+                }]
+            },
+            {
+                "type": "Questionnaire",
+                "id": "work-travel",
+                "title": "{{ [answers.first_name, answers.last_name] | format_household_name }}",
+                "description": "",
+                "questions": [{
+                    "title": "How do you usually travel to work?",
+                    "id": "work-travel-question",
+                    "type": "General",
+                    "answers": [{
+                        "id": "work-travel-answer",
+                        "mandatory": true,
+                        "guidance": {
+                            "show_guidance": "Show further guidance",
+                            "hide_guidance": "Hide further guidance",
+                            "content": [{
+                                    "description": "Select the option for the longest part, in distance, of your journey"
+                                },
+                                {
+                                    "description": "For example, you travel by bus and by car. Your bus journey is 5 miles, and the train journey is 8 miles. You should select train as your answer"
```

Comment:
@MIslam92 same issue with this description. bus, car, train?
this should be consistent with household